.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
**[Optional]**

- [ ] You are not using Github Flow which was requirement but you are using Gitflow. These two are two different things but in this project you are supposed to use [Github Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) 😊